### ISSUE TITLE
Git Blame: The floating box does not appear as expected

### ISSUE THREAD

**Issue by phtty** [solomonkinard/public#17]
I've noticed that the floating box for the extended intro page looks like this：
![image](https://github.com/user-attachments/assets/cb394c89-e840-457d-b89b-3cf19bf93938)
But after I downloaded the extension, the floating box only looked like this, and I looked for all the setting options, but none of them could be configured to do what the extension introduction page shows
![gitissue](https://github.com/user-attachments/assets/1ac2a84e-21c5-435a-b2c4-8651ffc303a3)
Please tell me how to configure it so that the floating box contains not only a summary of the git commit, but also the committer, the commit date, etc

**Comment by solomonkinard** [comment]
With the following configuration, for example, the last character (and first) of the line with the cursor on it will show the full commit, as shown in the screenshot.

```
  "git-blame.config.showBlameInline": true,
  "git-blame.config.inlineBlameHoverMessage": false,
  "git-blame.config.inlineBlame.hoverMessage.activeLine": true,
```

Do you want the full commit for every line, and not just the first and last character of the active line?

**Comment by phtty** [comment]
No, I'm not expecting the full commit to appear on every line. It's great now. It's just that after I set it up like this, it still only shows a small amount of commit information, and I think it might be a bug.
![033f8bc8a18b258eb568d946e251bd6a](https://github.com/user-attachments/assets/9e246d85-6a4a-4d5d-ab81-46e4916b3a45)
![05ed591f52440bd6af207c0f129e1909](https://github.com/user-attachments/assets/166e59e8-4551-4fcc-8d5e-4a6a905120df)

**Comment by solomonkinard** [comment]
There are three hover areas, and only two of them can show the full hover information currently.
1) first character of active line: can show the full commit message
2) last character of active line: can show the full commit message
3) blame hover after the last character of any line: does not show the full commit message.

I think that you want the full commit message for 3). If so, that is a feature that hasn't been implemented, rather than a bug. Do you want the full commit message for 3), and if so, that's a feature request? 1) and 2) already have the full commit message.

**Comment by phtty** [comment]
> There are three hover areas, and only two of them can show the full hover information currently.
> 
> 1. first character of active line: can show the full commit message
> 2. last character of active line: can show the full commit message
> 3. blame hover after the last character of any line: does not show the full commit message.
> 
> I think that you want the full commit message for 3). If so, that is a feature that hasn't been implemented, rather than a bug. Do you want the full commit message for 3), and if so, that's a feature request? 1) and 2) already have the full commit message.

I'm glad you replied so quickly, I really did try the hover area of 1 and 2, but neither of them showed the full commit information, and when I hover over the last character of the cursor line, it really only showed a partial commit message, and according to your description, it should show all the commit information. Also, while 3 isn't what I wanted, I think it's a cool feature.
